strings/base_reference_produce.h

```diff
@@ -377,6 +377,56 @@ WINRT_EXPORT namespace winrt
         return default_value;
     }
 
+    template <typename T>
+    auto try_unbox_value(Windows::Foundation::IInspectable const& value)
+    {
+        if constexpr (std::is_base_of_v<Windows::Foundation::IInspectable, T>)
+        {
+            return value.try_as<T>();
```

Comment:
TryLookup returns std::optional<T>{} when not present, yes?  Does that .try_as<T> return nullptr if the value is empty? Or does the `if (!value)` below need to move upwards?

Reply:
On TryLookup failure, it returns `nullptr` for reference types and `std:optional{}` for value types.

Reply:
Thanks, try_as of nullpter is what I didn't know.